python/lib/solver.cc

```diff
@@ -284,12 +314,12 @@ WrapLBS(py::module& slv)
       LBSSolverIO::WriteFluxMoments(self, file_base);
     },
     R"(
-    ???
+    Write flux moments to file.
 
     Parameters
     ----------
     file_base: str
-        ???
+        Filename prefix.
```

Comment:
I usually call this a basename like in linux.

Reply:
Fixed.